### DISCUSSION TITLE
Feeds never seem to update on the CRON schedule, but CRON is run and log is written

### DISCUSSION THREAD

**Discussion by marc0janssen** [FreshRSS/FreshRSS#5713] [Q&A] [answered]
**Describe the bug**
FreshRSS doesn't not seems to update the feed on the scheduled RSS, but the /tmp/FreshRSS.log is written.

**To Reproduce**
I have installed FreshRSS in the past 24 hours 4 times on three different docker systems.

1. IntelNUC(1) with Docker.
This was the first install from the freshrss/freshrss image. 
This one worked as a test if I could this software. Added few feeds, they worked, but I can't recall the scheduler work since I wanted a quick test on its features.

2. Raspberry PI3 with Docker.
This was an install from the freshrss/freshrss image. 
This one worked too, but here I explored the CRON_MIN option. It didn't seem to update the feeds, although the /tmp/FreshRSS.log was written at the aspected intervals.

Later that evening, suddenly with no clear change in the setup, **it started to work**

3. Local Cloudhoster
This was all setup the correct way, scheduler setup, /tmp/FreshRSS.log was written... no updates of the feeds.

4. IntelNUC(2) with Docker (different machine as in bullit 1, let's say I have a few running here ;-)
This is the Linuxserver.io Image.
This is setup correctly too.... Cron is running /tmp/FreshRSS.log is written, but the feeds are not being updated.

**Expected behavior**
On the intervals of the CRON, new feed-items are getting downloaded and shown as new-news-items in the list.

**Environment information (please complete the following information):**
- Intel NUC
root@uruk:/home/admin/docker/freshrss/scripts# cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=22.04
DISTRIB_CODENAME=jammy
DISTRIB_DESCRIPTION="Ubuntu 22.04.3 LTS"
- Browser: Safari macOS Sonoma 14.0
- FreshRSS version: 1.21.0
- Database version: ?? (no external database)
- PHP version: 
root@c2328f92902c:/tmp# php --version
PHP 8.2.10 (cli) (built: Aug 31 2023 20:34:52) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.2.10, Copyright (c) Zend Technologies

- Installation type: DOCKER Linuxserver.io

**Comment by marc0janssen** [comment]
Impossible... after 2 hours suddenly the IntelNUC install starts working also.... out of nothing...

There a explanation for this?

**Comment by Frenzie** [comment, marked as answer]
There's the maximum update frequency that's separate from the cron job. If it's not time to update yet the cron job will skip that feed.

**Comment by marc0janssen** [comment]
I appreciate your thorough explanation. It helped me understand the situation.

**Comment by marc0janssen** [comment]
By the way can i check that somewhere?

**Comment by Frenzie** [comment]
The default is under settings → Archiving and per feed… well, in the feed settings. ;-)

**Comment by marc0janssen** [comment]
Thanks. I understand it now.

**Comment by marc0janssen** [comment]
thanks for the help.
